src/berry_blender.c

```diff
@@ -1115,7 +1091,7 @@ static void CB2_LoadBerryBlender(void)
             sBerryBlender->mainState++;
         break;
     case 4:
-        if (PrintMessage(&sBerryBlender->textState, sText_BerryBlenderStart, GetPlayerTextSpeedDelay()))
+        if (PrintMessage(&sBerryBlender->textState, COMPOUND_STRING("Starting up the BERRY BLENDER.\pPlease select a BERRY from your BAG\nto put in the BERRY BLENDER.\p"), GetPlayerTextSpeedDelay()))
```

Comment:
But isn't this to much? By this logic we would have to turn everything into a compound string and for some reason I don't feel like it is a good idea. (follow up to a previous comment.)

Reply:
Yeah even when I was doing this commit, I felt like it was overkill. But I figured it wouldn't hurt to do the commit to get feedback and have to revert it